docs/getting_started.rst

```diff
@@ -3,24 +3,15 @@ Getting Started
 
 Now that your Raiden node is up and running, you can choose from the following tutorials, depending on your Raiden use case:
 
-Where do you want to use Raiden?
-================================
+See :doc:`Using Raiden on Mainnet <using-raiden-on-mainnet/overview>` for getting started to use Raiden in the Ethereum mainnet.
```

Comment:
I'd rather use "on the Ethereum mainnet" than "in the Ethereum mainnet". The former phrasing also seems to be used elsewhere, e.g. 
```
Using Raiden on Mainnet Guide <using-raiden-on-mainnet/overview.rst>
```

Reply:
Right, that would be consistent